README.md

```diff
@@ -8,3 +8,97 @@ Authors Haven - A Social platform for the creative at heart.
 ## Vision
 Create a community of like minded authors to foster inspiration and innovation
 by leveraging the modern web.
+
+## Application Overview
+There is plenty of media in the world already. No matter what happens to “traditional media”, there’s nothing to stop the torrent of information rushing from smartphones, corporations, and media startups onto the Internet, available for the world to see.
```

Comment:
Not particularly sure how these 3 bullets connect with the understanding of what the application is all about. Please provide clarity.

Reply:
I'll do that, thanks for the review and feedback.